tools/filetop.py

```diff
@@ -164,8 +174,13 @@
 
 # initialize BPF
 b = BPF(text=bpf_text)
-b.attach_kprobe(event="vfs_read", fn_name="trace_read_entry")
-b.attach_kprobe(event="vfs_write", fn_name="trace_write_entry")
+if not args.writeonly:
+    b.attach_kprobe(event="vfs_read", fn_name="trace_read_entry")
+if not args.readonly:
+    b.attach_kprobe(event="vfs_write", fn_name="trace_write_entry")
+if args.readonly and args.writeonly:
```

Comment:
read_only and write_only?

Reply:
I kept the flags this way to handle the cases when neither --read-only/--write-only are passed, or when both are passed. If you prefer I can make it throw an error when both are passed instead

Reply:
Let us do without --read-only/--write-only, both will handle. If both --read-only/--write-only, let us throw an error.

Reply:
Updated